src/test/java/io/stargate/sgv2/jsonapi/api/model/command/deserializers/FilterClauseDeserializerTest.java

```diff
@@ -78,6 +78,142 @@ ValueComparisonOperator.EQ, new JsonLiteral("aaron", JsonType.STRING))),
           .isEqualTo(expectedResult.getPath());
     }
 
+    @Test
+    public void gtComparisonOperator() throws Exception {
```

Comment:
Although duplication in test cases is not a huge problem, I think these 4 test methods could probably be shrunk by extracting parameterized test method, passing "$gt", `ValueComparisonOperator.GT` and so on?